### ISSUE TITLE
Installation issue after mac os update Catalina

### ISSUE THREAD

**Issue by RalfKellner** [rstudio/tensorflow#367]
Hey everyone,

I realized that during the new update, anaconda was transferred/deleted due to security issues. I removed and reinstalled my Anaconda distribution, R and RStudio. Now when I want to install tensorflow, the error I get is that "Tools for managing Python virtual environments are not installed". I suppose that R does not find the newly installed Python/Anaconda distribution. Can someone help here?

Thanks, 
Ralf

**Comment by dfalbel** [comment]
Hi Ralf, I am investigating this error, but I still coudn't reproduce it.
Can you point what's you `reticulate` version?

Related to: https://github.com/rstudio/keras/issues/898

**Comment by RalfKellner** [comment]
Hey Daniel, many thanks for your quick response. I am using reticulate version 1.13. Just an additional note. After reinstalling anaconda as explained on their webpage, the python interpreter lies on usr/usrname/opt/bin/python which is probably why I am not able to run any python code via retitculate. Unfortunately, setting the path to the interpreter via use_python() seems not to have an impact, I am getting an error 1 message from reticulate. Nevertheless, my problem seems to be related to this rather than to the install_tensorflow() function. Anyway, any help would be highly appreciated.

**Comment by dfalbel** [comment]
Hi Ralf,

The conda location shouldn't be a problem if it's available on your path - ie. you can run `conda list` in your terminal. Can you verify that?

**Comment by RalfKellner** [comment]
Hi Daniel,

yes you are right, it isn't. I could install tensorflow now, but still it is not working, but it seems to me that it is related to a general tensorflow problem. The error I get is:

Error in py_get_attr_impl(x, name, silent):
AttributeError: 'module' has no attribute ...

However, this does not only occur in R, but also in the terminal and via anaconda installation in the jupyter notebook. Are you aware of any issues with tensorflow 2.0? This is the version that gets currently installed. On another server I am using, we still have tensorflow 1.13. and everything works fine. 

Is there a way to install an older tensorflow version in R? I guess this could solve my problem.

**Comment by RimAboulaasri** [comment]
Hi @RalfKellner, 
I am facing a similar issue when I try to install and use Keras in RStudio. Here is the [issue](https://github.com/rstudio/keras/issues/909) I recently opened to ask for some help. I also get the error : 

Error in py_get_attr_impl(x, name, silent):
AttributeError: 'module' has no attribute ...

when I try to use any Tensorflow or Keras method in RStudio. 

Did you manage to solve the problem ?
Thank you, 
Rim

**Comment by RalfKellner** [comment]
@RimAboulaasri, kind of...I am convinced that this is not related to R and rather is an issue of tensorflow 2.0 and Mac OS Catalina. What I did is to install tensorflow 1.15 in the terminal and then tell R to use this installation via reticulate. But it was kind instable and I had to reinstall tensorflow 1.15 in another virtual environment and redirect R again to that environment...I still hope this issue gets soon fixed, because it is annoying. Shame on Catalina...that is not the only issue I have after updating.

Nevertheless, just working with python 3.7.3. and Tensorflow 1.15., everything works fine
